src/main/java/it/project1/monster/MonsterEntity.java

```diff
@@ -1,21 +1,21 @@
 package it.project1.monster;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
+import jakarta.persistence.*;
 
+@Entity
+@Table(name = "monsters")
 public class MonsterEntity {
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "monster_id")
```

Comment:
credo che per integrità, dato che anche noi abbiamo fatto così, quando crei una entity, ti convenga dargli come attributo primary key soltanto il nome "id", in questo caso magari a chi serve l'id di un mostro, sa che può scrivere "monsters.id" e trovarlo, come nel caso se ti serva un account o un item puoi scrivere tranquillamente "accounts.id" o "items.id", credo sia solo una cosa di convenzione